Cargo.toml

```diff
@@ -39,7 +39,7 @@ futures = "0.3.30"
 env_logger = "0.11.1"
 rand = "0.8.5"
 time = { version = "0.3.31", features = ["macros"] }
-tokio = { version = "1.35.1", features = ["macros"] }
+tokio = { version = "1.35.1", features = ["macros", "rt-multi-thread"] }
```

Comment:
It would be helpful to add the description of this PR here. Otherwise the decision for choosing `rt-multi-threaded` over `rt` is out of sight.